### ISSUE TITLE
The inference effect is very poor

### ISSUE THREAD

**Issue by xiewende** [woshidandan/TANet-image-aesthetics-and-quality-assessment#26]
When I use the pre-trained weights（SRCC_758_LCC_765.pth） you provided for inference, I get the following results:
test, accuracy: 0.8004852849092048, lcc_mean: 0.7643782940100576, srcc_mean: 0.7552650871245324, validate_losses: 0.04839770108433388
Why is the accuracy only 0.8, while the paper provides 0.94?
